pulp_ansible/app/tasks/utils.py

```diff
@@ -12,7 +12,7 @@
 
 def get_api_version(url):
     """Get API version."""
-    result = re.findall(r"/api/v(\d)", url)
+    result = re.findall(r"/v(\d)/", url)
     if len(result) != 1:
```

Comment:
Actually I think this should be `if len(result) == 0` because in the case when there is a v2 in the collection name this will be 2+.